### ISSUE TITLE
"entered unreachable code" in keypair-generator

### ISSUE THREAD

**Issue by frol** [nearprotocol/nearcore#1559]
If I don't specify the command (`signer-key` / `node-key`) I get a panic:

```
$ cargo run -p keypair-generator -- --generate-config --account-id=frol.near
thread 'main' panicked at 'internal error: entered unreachable code', genesis-tools/keypair-generator/src/main.rs:89:19
```

**Comment by ailisp** [comment]
The near cli has a similar issue before and fixed in #1272. I can quickly fix this

**Comment by bowenwang1996** [comment]
@ailisp I am already working on this as well as some other issues.
